CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [1.1.0]  - 2023-07-17
+
+
+### Changed
+- Added the EIC endpoint
+- Added env condition for infra
+- Added the automation script
+- Updated the Architecture Diagram in the Readme file
+- Updated the Readme file
+
+
+### Fixed
+- Linked List Lambda issue fixed
+- Fixed some issues in the pre-req.sh bash script
```

Comment:
Mention the issues as well

Reply:
Added in 51cb5a12dd43a8abb2932db2fa4de5d270cb1edf